pkg/wasm/client.go

```diff
@@ -0,0 +1,63 @@
+//go:build wasm
```

Comment:
Does this have to be it's own package?

Can we just have this be `pkg/internal/client/client_wasm.go` and `cmd/zed-wasm/main.go`?

Reply:
I mean, we could, but this matches how we define things in SpiceDB for WASM as well

Reply:
Honestly? I prefer having a distinct package. It keeps all the WASM-related stuff in one place and doesn't pollute the other namespaces. As well, this isn't really a "cmd", per se